### PR TITLE
Clean up obsolete Jenkinsfile branches, fix vendored flow

### PR DESCRIPTION
**Pull request by chromakode**
:eyeglasses: @keybase/react-hackers

### CODE REVIEW

**Comment by chromakode** [comment]
cc @jzila

File: Jenkinsfile [hunk 1]
```diff
@@ -33,7 +33,8 @@ node("ec2-fleet") {
             //],
     ])
 
-    env.GOPATH=pwd()
+    env.BASEDIR=pwd()
+    env.GOPATH="${env.BASEDIR}/go"
```
Comment (jzila) [inline Jenkinsfile]: this also needs to happen on the windows and osx nodes

**Comment by jzila** [comment]
lgtm once the fix is on windows and osx too

File: Jenkinsfile (hunk 1 above)
Comment (chromakode) [inline Jenkinsfile]: Damn, can't believe I missed that! Fixed. :)

**Comment by chromakode** [comment]
:white_check_mark: continuing in original branch

**Pull request closed without merging**